sources/packages/backend/apps/api/src/utilities/system-configurations-constants.ts

```diff
@@ -81,3 +81,8 @@ export const OFFERING_BULK_UPLOAD_MAX_FILE_SIZE = 4194304;
  * Expected length of the award value code.
  */
 export const AWARD_VALUE_CODE_LENGTH = 4;
+
+/**
+ * Minium unsuccessful weeks.
+ */
+export const MINIMUM_UNSUCCESSFUL_WEEKS = 68;
```

Comment:
Minimum unsuccessful weeks for what? Could it be more specific, please?

Reply:
The variable was just moved, not introduced in this PR, but I see the point and I will change it.

Reply:
Comments reviewed.